components/system/components/CodeBlock.js

```diff
@@ -54,7 +58,7 @@ export class CodeBlock extends React.Component {
           return (
             <div css={STYLES_PADDING}>
               <div css={STYLES_PRE}>{index}</div>
-              <div css={STYLES_CODE}>{element}</div>
+              <div css={STYLES_CODE}><pre><code className="language-javascript">{element}</code></pre></div>
```

Comment:
I'm glad this works, but we should be able to remove a couple of elements here and fix formatting.